PCbuild/prepare_ssl.py

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env python3
+# -*- coding: utf8 -*-      for <2.7>/PC/VS9.0/build_ssl.py
```

Comment:
The recommanded format is "# -*- coding: <encoding-name> -*-", not "# -*- coding: <encoding-name> -*-      for <2.7>/PC/VS9.0/build_ssl.py". I'm concerned by the trailing comment.